spec/unit/warehouse_spec.rb

```diff
@@ -4,19 +4,19 @@
   let(:empty_warehouse) { Warehouse.new(product_id: nil, amount: nil) }
   let(:warehouse) { Warehouse.new(product_id: 1, amount: 10) }
 
-  it "has 2 params" do
+  it "Warehouse should have 2 params" do
```

Comment:
Previous name was better, Warehouse is not necessary because it is test for warehouse, and we don't use should in test description